### PR TITLE
Grid patterns file for lesson 2

### PR DESCRIPTION
**Pull request by avalvillar**
I also have a text file of testing of this grid pattern file. Unsure if it should be included in submission for reference. Left out for now.

### CODE REVIEW

**Comment by avalvillar** [comment]
I also have a text file for the output of FizzBuzz 1-100 inclusive.

**Comment by avalvillar** [comment]
Added the Series file for lesson 2.

File: lesson02/fizz_buzz.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"""
+	Author is Antonio. V. Alvillar
+	Self-Paced-Online Python 210 UWPCE
+	November 6th 2018
+"""
+#Fizz Buzz problem
+def fizzBuzz():
```
Comment (charlesrobison) [inline lesson02/fizz_buzz.py]: You'll want your function names to be snake_case per PEP8

File: lesson02/fizz_buzz.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    for i in range(1,101): # 1-100 inclusive
+        if (i % 3 == 0 ) and (i % 5 == 0): #if multiple of 3 and 5
+            print("FizzBuzz")
+        elif (i % 3 == 0): #if multiple of 3 but not 5
+            print("Fizz")
+        elif (i % 5 == 0): #if multiple of 5 but not 3
+            print("Buzz")
+        else: #if not multiple of 3 or 5
+            print(i)
```
Comment (charlesrobison) [inline lesson02/fizz_buzz.py]: Works well!

File: lesson02/grid_patterns.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            spaceCount = 0
+        while dashCount < boxes: # loop for the box outlines
+            print('+' + '-' * size, end='')
+            dashCount += 1
+        print('+') # fencepost
+        #reinitialize variables and increment column/row (boxes)
+        colrow += 1
+        lineBreak = 0
+        dashCount = 0
+        spaceCount = 0
```
Comment (charlesrobison) [inline lesson02/grid_patterns.py]: Works well!

File: lesson02/series.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+#Assert Statements: Will display alert message if expected output is incorrect
+#Fibonacci
+assert fibonacci2(0) == 0, "fibonacci(0) = 0 FAILED"
+assert fibonacci2(5) == 5, "fibonacci(5) = 5 FAILED"
+assert fibonacci2(7) == 13, "fibonacci(7) = 13 FAILED"
+#Lucas
+assert lucas2(0) == 2, "lucas(0) = 2 FAILED"
+assert lucas2(3) == 4, "lucas(3) = 4 FAILED"
+assert lucas2(6) == 18, "lucas(6) = 18 FAILED"
```
Comment (charlesrobison) [inline lesson02/series.py]: Looks good - nice assert statements!

**Pull request merged**